Code/C++/Postfixexpression.cpp

```diff
@@ -0,0 +1,113 @@
+//Evaluation of Postfix expression
+
+/*A postfix expression is a combination of operands and operators in which the operators are placed after the operands.
+In the Evaluation of the postfix expression, we are given a string which contains postfix expression.We have to evaluate that expression.
+The main idea of the evaluation of postfix expression is as follows:
+While reading the expression from left to right we should push the element in the stack if it is an operand.
+Pop the two operands from the stack, if the element is an operator and then evaluate it.
+Push back the result of the evaluation. Repeat it till the end of the expression.
+*/
+
+#include <iostream>
+#include <math.h>
+using namespace std;
+
+class stack {
+  int top;
+  float stk[50];
+  public:
+    stack() {
+      top = -1;
+    }
+  void push(float p) {
+    stk[++top] = p;
+  }
+  float pop() {
+    return stk[top--];
+  }
+};
+
+int main()
+
+{
+  char p[50];
+  float data[50], a, b;
+  stack st;
+  cout << "Enter Postfix Expression with out any blank spaces:";
+  cin >> p;
+  int i = 0;
+  while (p[i] != '\0') {
+    if (isalpha(p[i])) {
+      cout << "Enter values of" << p[i] << ":";
+      cin >> data[i];
+    }
+    i++;
+  }
+  p[i] = ')';
+  i = 0;
+  while (p[i] != ')') {
+    switch (p[i]) {
+    case '+':
+      a = st.pop();
+      b = st.pop();
+
+      st.push(b + a);
+      break;
+    case '-':
+      a = st.pop();
+      b = st.pop();
+      st.push(b - a);
+      break;
+    case '*':
+      a = st.pop();
+      b = st.pop();
+      st.push(b * a);
+      break;
+    case '/':
+      a = st.pop();
+      b = st.pop();
+      st.push(b / a);
+      break;
+    case '^':
+      a = st.pop();
+      b = st.pop();
+      st.push(pow(b, a));
+      break;
+    default:
+      st.push(data[i]);
+    }
+    i++;
+  }
+  p[i] = '\0';
+  cout << "The given postfix expression is:";
+  for (i = 0; p[i]; i++)
+
+  {
+    if (isalpha(p[i]))
+      cout << data[i] << " ";
+    else
+      cout << p[i] << " ";
+  }
+  cout << "\nThe result of evaluated postfix expression is:" << st.pop();
+}
+
+/* Test cases:
+1.
```

Comment:
Plz follow the format of :
Input:
Output:

Reply:
Hi
I have done  the requested changes. Pls review it and if any further changes are required I would be happy to do them.